src/commands/app/create.ts

```diff
@@ -88,7 +92,8 @@ export default class AppCreate extends Command {
             .filter(
               (plan) =>
                 plans.projects[plan].available &&
-                plans.projects[plan].region === "iran"
+                plans.projects[plan].region === "iran" &&
+                (['react', 'angular', 'vue', 'static'].includes(platform) || plans.projects[plan].price > 0)
```

Comment:
You can use `FREE_PLAN_PLATFORMS` constant here.

Reply:
@vashian  did it on #62